src/VisualStudio/LiveShare/Impl/AbstractGoToDefinitionWithFindUsagesServiceHandler.cs

```diff
@@ -54,7 +54,7 @@ public async Task<object> HandleAsync(LSP.TextDocumentPositionParams request, Re
             if ((locations.Length == 0) && document.SupportsSemanticModel && this._metadataAsSourceService != null)
             {
                 var symbol = await SymbolFinder.FindSymbolAtPositionAsync(document, position, cancellationToken).ConfigureAwait(false);
-                if (symbol?.Locations.FirstOrDefault().IsInMetadata == true)
+                if (symbol?.Locations.FirstOrDefault()?.IsInMetadata == true)
```

Comment:
💭 It would be nice if it could have something like this:

```suggestion
                if (symbol?.Locations.FirstOrDefault(Location.None).IsInMetadata == true)
```

Reply:
It is not, and I don't believe it would actually change the effect of the code. Please let me know if you think it would.

Reply:
It wouldn't change behavior